### PR TITLE
Fix filename

### PR DESCRIPTION
**Pull request by mrdadah**

### CODE REVIEW

File: src/components/DatasetEditor/EditableTitleText.js [hunk 1]
```diff
@@ -67,6 +32,7 @@ export default ({ value, onChange }) => {
       InputProps={{
         inputProps: { style: { color: "#000" } },
       }}
+      onBlur={handleBlue}
```
Comment (seveibar) [inline src/components/DatasetEditor/EditableTitleText.js:35]: minor typo: `handleBlue` -> `handleBlur`. fwiw I would avoid the word "handle" which can imply state. `onBlur` is fine for the name of a function (even though it's also the name of the prop).

File: src/components/DatasetEditor/EditableTitleText.js [hunk 2]
```diff
@@ -10,53 +10,18 @@ const useStyles = makeStyles({
   },
 })
 
-const SAVE_WAIT = 2000
-
 export default ({ value, onChange }) => {
```
Comment (seveibar) [inline src/components/DatasetEditor/EditableTitleText.js:13]: nice, didn't know if you'd be able to do this and keep it controlled but looks simple

File: src/components/DatasetEditor/index.story.js [hunk 3]
```diff
@@ -92,3 +93,21 @@ storiesOf("DatasetEditor", module)
...
+      </div>
+      <EditableTitleText
+        label="File Name"
+        onChange={(newName) => {
+          setValueDisplay(newName)
+        }}
+        value={valueDisplay || ""}
+      />
+    </>
+  )
```
Comment (seveibar) [inline src/components/DatasetEditor/index.story.js:112]: nice!

**Review by seveibar — APPROVED** [review]
awesome PR, great simplification, let's fix the minor typo and merge?

File: src/components/DatasetEditor/EditableTitleText.js (hunk 1 above)
Comment (mrdadah) [reply src/components/DatasetEditor/EditableTitleText.js]: yes, makes sense

File: src/components/DatasetEditor/EditableTitleText.js (hunk 2 above)
Comment (mrdadah) [reply src/components/DatasetEditor/EditableTitleText.js:13]: yes, I added story to check the changed values for the parent component and it's working fine for controlled component.

you can double check as well

**Review by seveibar — APPROVED** [review]

**Pull request merged**